app/concepts/matestack/ui/core/bdo/bdo.rb

```diff
@@ -0,0 +1,11 @@
+module Matestack::Ui::Core::Bdo
+  class Bdo < Matestack::Ui::Core::Component::Static
+    REQUIRED_KEYS = [:dir]
```

Comment:
Since it throws an exception if no `dir` attribute is provided it could make sense to add a test for that.

Reply:
I agree, maybe if you can add it in as one more test case... Just write one where it expects to fail and then I can approve of this to be merged!